src/views/instructor/training/EditSession.vue

```diff
@@ -173,6 +177,26 @@ export default {
 				console.log(e);
 			}
 		},
+		async submitTraining() {
+			try {
+				const formData = new FormData();
+				formData.append('cid', this.session.cid);
+				formData.append('session_date', this.session.startTime);
+				formData.append('position', this.session.position);
+				formData.append('duration', duration);
+				formData.append('movements', this.session.movements);
+				formData.append('score', this.session.progress);
+				formData.append('notes', this.session.studentNotes);
+				formData.append('location', this.session.location);
+				formData.append('ots', this.session.ots);
+				formData.append('expDate', this.$refs.expirationDate.value);
+				await vatusaApiAuth.post(`/user/${this.session.cid}/training/record`, formData);
+
+				this.toastSuccess('Traning Record issued');
```

Comment:
Should be ''Training Session Submitted". Or "Training Notes Submitted"

Any preference here @JWZABARTCC ?